docs/home/native-features.md

```diff
@@ -0,0 +1,114 @@
+?> **Note:** Stay tuned for new feature instructions as we update and expand this page.
+
+# Integrating Native Features
+
+One of the best ways to upgrade your progressive web app is to take advantage of web capabilites to integrate with the user's operating system. Modern web technology has enabled a whole host of ways to make your PWA behave more like a native application and interact seamlessly with the OS.
+
+This article will showcase how to enable various native functionality for your progressive web app.
+
+## Shortcuts 
+
+Application shorcuts allow your user to directly navigate to certain parts of your progressive web app directly from the operating system. When shorctuts are enabled, they can be accessed through your application's context menu. In Windows, the context menu can be opened by right-clicking on your app's icon.
+
+A context menu with shortcuts will look like this:
+
+<div class="docs-image">
+   <img src="assets/home/native-features/shortcuts.png" alt="The shortcuts menu open in Windows." width=600>
+</div>
+
+Clicking a specific shortcut will take you directly to the associated content within your PWA. For example, in the screenshot above, clicking "PWA Starter" would open the PWABuilder Documentation PWA and navigate directly to the documentation for the PWA Starter.
+
+### How to Implement Shortcuts
+
+All thats necessary to add shortcuts to your progressive web app is adding a valid `shortcuts` field to your web app manifest. The `shortcuts` field accepts a list of `shortcut` objects and a very basic example would look like this: 
+
+```json
+"shortcuts": [
+  {
+    "name": "News Feed",
+    "url": "/feed",
+    "description": "Noteworthy news from today."
+  },
+  {
+    "name": "New Post",
+    "url": "/post",
+    "description": "Create a new post."
+  }
+]
+```
+
+These shortcut objects just have a `name`, a `url` to navigate to, and a brief `description` of the shorcut.
+
+There are a few extra fields you can add to spruce up your shortcut. Another example:
+
+```json
+"shortcuts": [
+  {
+    "name": "News Feed",
+    "short_name": "Feed",
+    "url": "/feed",
+    "description": "Noteworthy news from today.",
+    "icons": [
+      {
+        "src": "assets/icons/news.png",
+        "type": "image/png",
+        "purpose": "any"
+      }
+    ]
+  }
+]
+```
+
+In this example, we've added a few extra fields to our shorcut. `short_name` provides an alternate name to display for your shortcut if there is limited display space, while `icons` allows you to specify a custom icon to display for your shorcut in the context menu.
```

Comment:
It's important to note that if you are adding an icons array, you must have a 96x96 icon in there. From MDN, _"A set of icons that represent the shortcut. They can be used, e.g., in the context menu. When included, the icon set must include a 96x96 pixel icon."_

Reply:
```suggestion
In this example, we've added a few extra fields to our shorcut. `short_name` provides an alternate name to display for your shortcut if there is limited display space, while `icons` allows you to specify a custom icon to display for your shorcut in the context menu. For shortcuts, the array you provide must include an icon that is 96x96.
```

Reply:
```suggestion
In this example, we've added a few extra fields to our shorcut. `short_name` provides an alternate name to display for your shortcut if there is limited display space, while `icons` allows you to specify a custom icon to display for your shorcut in the context menu. For shortcuts, the array you provide must include an icon that is 96x96.
```